mappings/servers/akyto/metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+    "id": "akyto",
+    "name": "akyto",
+    "description": "The return of the real fight is here.",
+    "addresses": [
+        "eu.akyto.club",
+        "na.akyto.club"
```

Comment:
No need to specify subdomains. There's only need for more than one if you have multiple domains registered